### ISSUE TITLE
Question Regarding Poor Performance on ScanNet After Code Reproduction

### ISSUE THREAD

**Issue by haopengmai** [prs-eth/Marigold#116]
I encountered unexpectedly poor performance when testing on the ScanNet dataset. I have not made any modifications to the original code, yet the results differ significantly from what I anticipated.

Could you kindly advise if there are any particular settings or configurations that I might have overlooked? 

Thank you very much for your time and assistance. 
<img width="1200" alt="image" src="https://github.com/user-attachments/assets/55ca6a20-34af-468c-8209-91ea3eb1ffa0">

**Comment by markkua** [comment]
Hi, did you follow the instructions [here](https://github.com/prs-eth/Marigold?tab=readme-ov-file#-evaluation-on-test-datasets-) to run the evaluation? Could you double-check which checkpoint you are using and share the command that you ran? The log would also be helpful.

**Comment by haopengmai** [comment]
![836058668](https://github.com/user-attachments/assets/6b9a47a3-4f37-412e-bbcb-b828b63986d9)

I have double-checked my procedure and followed the provided command for evaluation. The only modification I made was replacing the UNet network from Stable Diffusion V2 with the best-performing UNet network. I ensured that the checkpoint path is correct, and the script command I ran is shown in the attached image.

Additionally, there were no errors in the log output during the execution of the script.

Thank you for taking the time to answer my question!~
